development.html

```diff
@@ -252,6 +266,7 @@ <h2 class="title">Visit the library</h2><!-- Change this Heading Title to your c
 // v2.9 Adding pipes between custom links in search result list (CSS only)
 // v3.0 Reworking recording of returned results, and clicks thereupon
 // v3.1 Removing red "beta-banner" bar as a test
+// v3.2 Switches to new Google Analytics script
```

Comment:
If we're going to do this with GitHub, then yes - I think this would be better done in GitHub.